hack/components/bump-linux-bridge.sh

```diff
@@ -27,10 +27,17 @@ echo 'Build container image with linux-bridge binaries'
 LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/github.com/containernetworking/plugins/bin
 LINUX_BRIDGE_IMAGE=quay.io/kubevirt/cni-default-plugins
 LINUX_BRIDGE_IMAGE_TAGGED=${LINUX_BRIDGE_IMAGE}:${LINUX_BRIDGE_TAG}
-(
-    cd ${LINUX_BRIDGE_PATH}
+ARCH=$(uname -m | sed 's/x86_64/amd64/')
+PLATFORMS="linux/amd64,linux/s390x"
```

Comment:
please add auto detection instead hardcoding both, we should be able to compile as fast as possible

Reply:
Done.